### PR TITLE
Write total neighbors to output

### PR DESCRIPTION
**Pull request by streeve**

### CODE REVIEW

File: src/CabanaPD_Solver.hpp [hunk 1]
```diff
@@ -158,7 +166,7 @@ class SolverElastic
         if ( print )
         {
             log( std::cout, "Local particles: ", particles->n_local,
-                 ", Maximum local neighbors: ", max_neighbors );
+                 ", Maximum neighbors: ", max_neighbors );
```
Comment (pabloseleson) [inline src/CabanaPD_Solver.hpp:169]: @streeve : Why do we have both here and in Line 185 the output of "Maximum neighbors: ", max_neighbors "?

File: src/CabanaPD_Solver.hpp [hunk 2]
```diff
@@ -148,8 +148,16 @@ class SolverElastic
         neighbors = std::make_shared<neighbor_type>( x, 0, particles->n_local,
                                                      force_model.delta, 1.0,
                                                      mesh_min, mesh_max );
-        int max_neighbors =
+        unsigned max_neighbors;
```
Comment (pabloseleson) [inline src/CabanaPD_Solver.hpp:151]: @streeve : Is this line kind of initialization? (same for Line 154)

Comment (streeve) [reply src/CabanaPD_Solver.hpp:151]: Yes

File: src/CabanaPD_Solver.hpp (hunk 1 above)
Comment (streeve) [reply src/CabanaPD_Solver.hpp:169]: This prints to screen, the other prints to file

**Review by pabloseleson — APPROVED** [review]

**Pull request merged**